I.write-tests/05-get-second-third-smallest/get-second-third.test.js

```diff
@@ -1,5 +1,11 @@
 // example
-// input = [90, 5, 11, 8, 6];
+// input = [4, 10, 32, 9, 21];
 // expected output = [6, 8];
 
-// also test that the original array has not changed
\ No newline at end of file
+// also test that the original array has not changed
+test("sort the array", function(){
+    var getSecondThird = require('./get-second-third');
+    var array = [90, 5, 11, 8, 6];
+    var result  = getSecondThird(array);
+    expect(result).toEqual(8,6);
```

Comment:
As a result of the change to the code you're testing, this test would actually pass if you just wrote `.toEqual(8)`. The second argument to `toEqual` is ignored, and the function only returns a single value.